tests/utils/tools.php

```diff
@@ -88,6 +88,15 @@ function get_module_info($row)
     return $matches[1];
 }
 
+function create_test_manager(string $uri = null, array $options = [], array $driverOptions = [])
+{
+    if (getenv('API_VERSION') && ! isset($driverOptions['serverApi'])) {
+        $driverOptions['serverApi'] = new \MongoDB\Driver\ServerApi(getenv('API_VERSION'));
```

Comment:
Consider a `use` statement at the top, as we do for other classes.

Reply:
Fixed